forc/src/cli/commands/new.rs

```diff
@@ -0,0 +1,35 @@
+use crate::ops::forc_new;
+use anyhow::Result;
+use clap::Parser;
+
+const TEMPLATE_HELP: &str = r#"Initialize a new project from a template.
+
+Example Templates: 
+ - counter"#;
+
+/// Create a new Forc project.
+#[derive(Debug, Parser)]
+pub struct Command {
+    /// Initialize a new project from a template
+    #[clap(short, long, help = TEMPLATE_HELP)]
+    pub template: Option<String>,
```

Comment:
The `template` option can be removed as #1614 has introduced a dedicated command for templates.